### PR TITLE
fix(jolt-core): Fixes DIVU and REMU sequence_output

### LINKED ISSUE #499: DIVU and REMU `sequence_output` incorrectly handle 0 divisor
https://github.com/a16z/jolt/issues/498#issuecomment-2463692225

### PR DESCRIPTION
**Pull request by mw2000**
We return the correct output from sequence output when the divisor is zero, for divu and remu.

Should fix #499

### CODE REVIEW

**Comment by mw2000** [comment]
From #498 I believe we just need to fix the sequence output (which is used in the jolt virtual sequence test macro) to be output uint::max when the divisor is zero.

File: jolt-core/src/jolt/instruction/remu.rs [hunk 1]
```diff
@@ -206,10 +206,20 @@ impl<const WORD_SIZE: usize> VirtualInstructionSequence for REMUInstruction<WORD
...
-            _ => panic!("Unsupported WORD_SIZE: {}", WORD_SIZE),
+        let quotient = if y == 0 {
+            match WORD_SIZE {
+                32 => u32::MAX as u64,
+                64 => u64::MAX,
+                _ => panic!("Unsupported WORD_SIZE: {}", WORD_SIZE),
+            }
+        } else {
+            x / y
+        };
```
Comment (moodlezoup) [inline jolt-core/src/jolt/instruction/remu.rs:217]: let's get rid of this and just move the existing `match` into the `else` block below

Comment (mw2000) [reply jolt-core/src/jolt/instruction/remu.rs:217]: Sounds good!

Comment (mw2000) [reply jolt-core/src/jolt/instruction/remu.rs]: Fix up

**Review by moodlezoup — APPROVED** [review]
thanks Mihir!

**Pull request merged**